examples/mother/Cargo.toml

```
@@ -0,0 +1,38 @@
+[package]
+name = "mother"
+description = "Mother of all contracts"
+version = "3.0.1"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2021"
+publish = false
+
+[dependencies]
+ink_primitives = { version = "3.0.1", path = "../../crates/primitives", default-features = false }
+ink_metadata = { version = "3.0.1", path = "../../crates/metadata", default-features = false, features = ["derive"], optional = true }
+ink_env = { version = "3.0.1", path = "../../crates/env", default-features = false }
+ink_storage = { version = "3.0.1", path = "../../crates/storage", default-features = false }
+ink_lang = { version = "3.0.1", path = "../../crates/lang", default-features = false }
+ink_prelude = { version = "3.0", path = "../../crates/prelude", default-features = false }
```

OR just remove the versions all together. There is no benefit of having them.

```suggestion
ink_prelude = { version = "3.0.1", path = "../../crates/prelude", default-features = false }
```